doc/md/guides/mysql/cd-101.md

```diff
@@ -0,0 +1,131 @@
+ALTER TABLE pets DROP FOREIGN KEY owner_id;
+```
+
+While this is sometimes desired, dropping a foreign key can have negative and sometimes unpredictable side effects which should be taken into consideration.
+Atlas's [`migrate lint`](/versioned/lint) command can be used to detect when a foreign key is about to be dropped. For example:
+```bash
+atlas migrate lint --dir file://migrations --dev-url "mysql://root:pass@localhost:3306/dev" --latest 1
+```
+If the file contains a foreign key drop, the output will be something like:
+
+```bash
+20230305125128.sql: constraint deletion detected:
+
+	L1: Dropping foreign-key constraint "owner_id"
+```
+
+
+## Impact of dropping foreign key constraints
+
+### Losing referential integrity
+Foreign keys are the means by which a relational database maintains the [Referential Integrity](https://en.wikipedia.org/wiki/Referential_integrity) of the data it stores.  In short, this means that a database can enforce that all references that it contains are valid. 
+
+For this reason, dropping a foreign key constraint should be done after considering the implication of no longer being able to guarantee referential integrity in the database.
+
+Suppose our database schema contains a foreign key constraint between the `pets` and `users` tables:
+<p style={{textAlign: "center"}}><a href="https://gh.atlasgo.cloud/explore/1b14810d"><img src="https://atlasgo.io/uploads/users-pets.png" alt="Pets ERD"/></a></p>
+
+As long as this constraint exists, it is not possible to insert records into the `pets` table that refer to a non-existing record in the `users` table. If we try to insert such a record
+
+.. INSERT QUERY ..
+
+Our statement will fail with the following error:
+
+.. error message ..
+
+We can drop the foreign key between `pets` and `users` with the following command: 
+
+```sql
+ALTER TABLE pets DROP FOREIGN KEY owner_fk;
+```
+
+Now, inserting data into the `pets` table is still possible, however it is also possible to insert "invalid" data. For example, assuming the `users` table is empty, the following command will not fail:
+
+```sql
+INSERT INTO pets (name, owner_id) VALUES ('Fortuna', 1);
+```
+
+Losing referential integrity can result in an unexpected impact on a system. For example, a query containing a`JOIN` statement that was previously guaranteed to return a result, may not return one in some cases:
+```sql
+SELECT
+    pets.name,
+    users.name AS owner_name
+FROM pets JOIN users ON pets.owner_id = users.id
+WHERE pets.name = "spidey";
+```
+After dropping the constraint, any code that relied on the invariant that pets are always linked to an existing owner may suddenly begin to fail.
+
+### Irreversible changes
+
+It is also important to note that dropping a foreign key constraint is not always a reversible operation. Irreversible operations increase the risk of any change. This is because if something unexpected happens, it will not be possible to bring the system to its previous stable state. 
```

Comment:
```suggestion
It is also important to note that dropping a foreign key constraint is not always a reversible operation. Irreversible operations increase the risk of any change. This is because if something unexpected happens, it is not possible to bring the system to its previous stable state. 
```